target/linux/x86/base-files/etc/board.d/01_leds

```diff
@@ -11,7 +11,7 @@ cisco-mx100-hw)
 	ucidef_set_led_usbport "usb" "USB" "mx100:green:usb" "1-1-port2"
 	ucidef_set_led_default "diag" "DIAG" "mx100:green:tricolor" "1"
 	;;
-pc-engines-apu1|pc-engines-apu2|pc-engines-apu3)
+pc-engines-apu1|pc-engines-apu2|pc-engines-apu3|pc-engines-apu4|pc-engines-apu6)
```

Comment:
Shouldn't we add v5 here too?

Reply:
Yes, good point.  Just need to confirm that it has 3 Ethernet ports.

Reply:
Actually, the APU5 only has two ethernet ports, so adjusting...